lua/cmp/entry.lua

```diff
@@ -223,7 +223,6 @@ entry.get_view = function(self, suggest_offset, entries_buf)
       view.menu.text = item.menu or ''
       view.menu.bytes = #view.menu.text
       view.menu.width = vim.fn.strdisplaywidth(view.menu.text)
-      view.menu.hl_group = 'CmpItemMenu'
```

Comment:
Why remove this line?

Reply:
With the `CmpWindow`/`CmpBorderedWindow` distinction, `view` cannot report `hl_group` correctly. `view` doesn't know if it is a documentation window or completion window, thus doesn't know what highlight group to use.

Thankfully, this property is never referenced and so removing it required no further changes.

Reply:
? No. This `hl_group` is not related to the window highlight. This will treated as like as `v:completed_item.menu` field in completion menu text.

Reply:
What highlight group should it be? `CmpItemMenu` was replaced by `CmpWindow`/`CmpBorderedWindow`. 

I didn't see it referenced anywhere either… removing it didn't seem to change anything. I'll triple check

Reply:
It is used in custom_entries_view.lua.

The `CmpItem*` highlight group is not relate for this PR. Please don't remove them.

Reply:
Understood. Added back